### ISSUE TITLE
Can I use normalized "trellis" as probability distribution over classes for each sequence element?

### ISSUE THREAD

**Issue by koren-v** [tensorflow/addons#2418]
Hi, I use your [piece of code](https://github.com/tensorflow/addons/blob/v0.12.0/tensorflow_addons/text/crf.py#L374) in my model, but instead of hard labels, I need to know the probability of each class for each timestamp. So can I use normalized `trellis` as these distributions?

**Comment by howl-anderson** [comment]
Hi @koren-v , that probability you mentioned is the emission probability? 
if so, I think you can use `tfa.layers.CRF` to get the potentials (as logits). 

In following sample code:
```python
layer = tfa.layers.CRF(4)
decoded_sequence, potentials, sequence_length, chain_kernel = layer(inputs)
```
`potentials` is the thing you want.

----

> So can I use normalized trellis as these distributions?

I did not understand your question, can you rephrase it in detail?

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
